### PR TITLE
docs(cookbook): Add dark theme description

### PR DESCRIPTION
**Pull request by Inaam-Ur-Rehman**
# Overview

# What is it?

- [x] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Checklist:

- [x ] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ x] I have performed a self-review of my own code
- [x ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/cookbook/Theme Management/index.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
+##Problem
```
Comment (mhevery) [inline packages/docs/src/routes/docs/cookbook/Theme Management/index.mdx:1]: Missing front matter.

**Review by mhevery — CHANGES_REQUESTED** [review]
Also let me know when this is ready for full review.

**Review by Inaam-Ur-Rehman — COMMENTED** [review]
Updated the frontmatter

**Review by mhevery — APPROVED** [review]

**Review by Inaam-Ur-Rehman — COMMENTED** [review]
There is typo in the contributor name. Can I change it

**Comment by gioboa** [comment]
@Inaam-Ur-Rehman yes, you must create a new PR for that.

**Pull request merged**